### PR TITLE
Add PayPal contract renewal page

### PR DESCRIPTION
**Pull request by DominikaK**
Ping @johaho for review

### CODE REVIEW

File: docs/ecommerce_plugins/paypal_contract_renewal.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Renewing a PayPal contract
+
+PayPal has a built-in security mechanism that expires a customer contract (subscription) after one year.
+
+After this happens, you need to adding payment information again to renew the contract.
```
Comment (MagdalenaZuba) [inline docs/ecommerce_plugins/paypal_contract_renewal.md]: you need to add

File: docs/ecommerce_plugins/paypal_contract_renewal.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Step 1: Go to https://admin.yoochoose.net and log in with your email and password.
+
+![Login screen for admin.yoochoose.net](img/paypal_renew_step1.png)
+
+Step 2: Click "Edit contract data" at the top. (If you have a German account, the link will say "Vertragsdaten ändern").
+
+![Dashboard with the Edit contract data option](img/paypal_renew_step2.png)
+
+Step 3: A new window opens with the billwerk partner site. Billwerk is responsible for handling Personalization payments.
```
Comment (MagdalenaZuba) [inline docs/ecommerce_plugins/paypal_contract_renewal.md:17]: A new window in German opens

File: mkdocs.yml [hunk 3]
```diff
@@ -34,6 +34,7 @@ pages:
     - E-Commerce Plugins:
         - 'Magento2': 'ecommerce_plugins/magento2.md'
         - 'OXID': 'ecommerce_plugins/oxid.md'
+        - 'Renewing a PayPal contract': 'ecommerce_plugins/paypal_contract_renewal.md'
```
Comment (MagdalenaZuba) [inline mkdocs.yml]: PayPal contract renewal - it's a title in the menu I would avoid 'a'

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/ecommerce_plugins/paypal_contract_renewal.md (hunk 2 above)
Comment (DominikaK) [reply docs/ecommerce_plugins/paypal_contract_renewal.md:17]: @johaho, could you confirm that billwerk website is always in German, or do some customers have access to an English interface?

**Review by MagdalenaZuba — APPROVED** [review]

**Comment by ezpersonalization** [comment]
Hi @DominikaK @MagdalenaZuba,
the page is fine. Go for it

**Pull request merged**